charon/src/ast/builtins.rs

```diff
@@ -2,7 +2,7 @@
 //!
 //! **IMPORTANT**:
 //! When checking whether names are equal to one of the reference names below,
-//! we ignore the disambiguators (see [crate::names] and [crate::names_utils]).
+//! we ignore the disambiguators (see [crate::names] and [crate::names_utils](crate::ast::names_utils)).
```

Comment:
For this whole commit, I would prefer to fix the text too, not just the link:
```suggestion
//! we ignore the disambiguators (see [crate::names] and [crate::ast::names_utils]).
```

Reply:
oh perfect, you did that in the next commit

Reply:
I've just looked for all instances of `(crate::` and fixed all but one (about the ids macro) occurrence according to VSCode. Would you like me to also inline the links to hax objects?

Reply:
Sure!